charts/nri-kube-events/templates/deployment.yaml

```diff
@@ -79,6 +79,10 @@ spec:
             - name: config
               mountPath: /etc/newrelic-infra.yml
               subPath: newrelic-infra.yml
+          {{- if .Values.resources }}
```

Comment:
Implemented in this way it will be not possible to specify different resource requests and limits for forwarder and kube-events containers. 
In some cases, we did this "simplification" and we users asked later on how to set them up with different values

Reply:
I see. I updated the code to include specific forwarder resources settings

Reply:
@paologallinaharbur I tried ".Values.forwarder.resources" but always failed at "test chart upgrade". I am not sure what happened. So I revert it back to .Values.resources

Reply:
updated the PR as we discussed offline last week